pkg/http/core/search_test.go

```diff
@@ -0,0 +1,101 @@
+package core_test
+
+import (
+	"errors"
+	"net/http"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Search", func() {
+	Describe("#Search", func() {
+		accountsHeader := ""
+
+		BeforeEach(func() {
+			setup()
+			uri = svr.URL + "/search?pageSize=500&q=default&type=pod"
```

Comment:
where is uri being initialized? I don't see uri :=

Reply:
`core_test.go` where we hold global test variables and setup.